mlflow/genai/scorers/builtin_scorers.py

```diff
@@ -212,71 +257,44 @@ def groundedness():
                 "retrieved_context": [{"content": "Paris is the capital city of France."}],
             }
         ]
-        result = mlflow.genai.evaluate(data=data, scorers=[groundedness()])
+        result = mlflow.genai.evaluate(data=data, scorers=[groundedness])
     """
-    return _Groundedness()
-
 
-class _GuidelineAdherence(_BaseBuiltInScorer):
-    name: str = "guideline_adherence"
-    global_guidelines: Optional[list[str]] = None
-    required_columns: set[str] = {"inputs", "outputs"}
-
-    def update_evaluation_config(self, evaluation_config) -> dict:
-        # Metric name should always be "guideline_adherence" regardless of the custom name
-        config = deepcopy(evaluation_config)
-        metrics = config.setdefault(GENAI_CONFIG_NAME, {}).setdefault("metrics", [])
-        if "guideline_adherence" not in metrics:
-            metrics.append("guideline_adherence")
+    name: str = "groundedness"
+    required_columns: set[str] = {"inputs", "outputs", "retrieved_context"}
 
-        # If global guidelines are specified, add it to the config
-        if self.global_guidelines:
-            # NB: The agent eval harness will take multiple global guidelines in a dictionary format
-            #   where the key is the name of the global guideline judge. Therefore, when multiple
-            #   judges are specified, we merge them into a single dictionary.
-            #   https://docs.databricks.com/aws/en/generative-ai/agent-evaluation/llm-judge-reference#examples
-            global_guidelines = config[GENAI_CONFIG_NAME].get("global_guidelines", {})
-            global_guidelines[self.name] = self.global_guidelines
-            config[GENAI_CONFIG_NAME]["global_guidelines"] = global_guidelines
+    def __call__(
+        self, *, inputs: Any, outputs: Any, retrieved_context: list[dict[str, Any]]
+    ) -> Assessment:
+        """Evaluate groundedness of response against context."""
+        return super().__call__(inputs=inputs, outputs=outputs, retrieved_context=retrieved_context)
 
-        return config
+    def configure(self, *, name: str = "groundedness") -> "Groundedness":
+        """
+        Configure the scorer with additional parameters such as name.
```

Comment:
```suggestion
        Configure the scorer with a specified name.
```

additional parameters sounds like it takes a dict or multiple parameters.